debian/rules

```diff
@@ -1,103 +1,21 @@
 #!/usr/bin/make -f
 
-dist_release := $(shell lsb_release -cs)
-
-dh_extra_flags = -plandscape-common -plandscape-client
-
-include /usr/share/python3/python.mk
-ifeq (,$(py_sitename))
-	py_sitename = site-packages
-	py_libdir = /usr/lib/python$(subst python,,$(1))/site-packages
-	py_sitename_sh = $(py_sitename)
-	py_libdir_sh = $(py_libdir)
-endif
+export DH_ALWAYS_EXCLUDE=.git
+export DH_VERBOSE = 1
 
 package = landscape-client
 root_dir = debian/tmp
 SHAREDIR = /usr/share/landscape
 LIBDIR = /usr/lib/landscape
 
-revision = $(shell dpkg-parsechangelog | grep ^Version | cut -f 2 -d " "| cut -f 2 -d "-")
-
-landscape_common_substvars = debian/landscape-common.substvars
-landscape_client_substvars = debian/landscape-client.substvars
-
-build-arch: build
+%:
+	dh $@ --with python3 --buildsystem=pybuild
 
-build-indep: build
-
-build: build-stamp
-build-stamp:
-	dh_testdir
-	sed -i -e "s/^DEBIAN_REVISION = \"\"/DEBIAN_REVISION = \"-$(revision)\"/g" landscape/__init__.py
-	python3 setup.py build
+override_dh_auto_install:
+	dh_auto_install
 	make -C apt-update
-	touch build-stamp
-
-clean:
-	dh_testdir
-	dh_testroot
-	rm -f build-stamp
-	rm -rf build
-	make -C apt-update clean
-	dh_clean
-	sed -i -e "s/^DEBIAN_REVISION = .*/DEBIAN_REVISION = \"\"/g" landscape/__init__.py
-
-install: build
-	dh_testdir
-	dh_testroot
-	dh_prep
-	dh_installdirs
-
-	python3 setup.py install --root $(root_dir)/ $(py_setup_install_args)
+	install -D -o root -g root -m 755 debian/landscape-sysinfo.wrapper $(CURDIR)/$(root_dir)$(SHAREDIR)/landscape-sysinfo.wrapper
+	install -D -o root -g root -m 755 apt-update/apt-update $(CURDIR)/$(root_dir)$(LIBDIR)/apt-update
 
-	# Keep in mind that some installed files are defined in setup.py
-	install -D -o root -g root -m 755 debian/landscape-sysinfo.wrapper $(root_dir)$(SHAREDIR)/landscape-sysinfo.wrapper
-	install -D -o root -g root -m 755 apt-update/apt-update $(root_dir)$(LIBDIR)/apt-update
-
-binary-indep:
-# do nothing
-#
-binary-arch: build install
-	dh_lintian
-	dh_testdir
-	dh_testroot
-	dh_installdocs
-	dh_installman
-	dh_installchangelogs
-	dh_install --sourcedir debian/tmp/
+override_dh_systemd_enable:
```

Comment:
This change doesn't seem to have the desired effect (disabled/stopped by default). I don't think dh_systemd_enable is even used if I look at the resulting postinst script. However, this did, whether idiomatic or not, create a disabled service which only gets (re)started if landscape-config enabled it:

override_dh_installsystemd:                                                     
  dh_installsystemd --no-enable --no-start

Reply:
You are right, good catch here:

## dh_systemd_enable (from debhelper 12)
```
if (not compat(10)) {
        error("dh_systemd_enable is no longer used in compat >= 11, please use dh_installsystemd instead");
```

Reply:
Fixed in my latest push